w3c-tag-advice-for-polyfill-authors.md

```diff
@@ -0,0 +1,94 @@
+# TAG Advice for polyfill authors
+
+The table below is to document the extent of which the MapML polyfill
+conforms to the W3C <abbr title="Technical Architecture Group">TAG</abbr>'s
+[Advice for polyfill authors](https://www.w3.org/2001/tag/doc/polyfills/#advice-for-polyfill-authors).
+
+<table>
+  <tr>
+    <th scope="col">TAG Advice</th>
+    <th scope="col">Conforms</th>
+    <th scope="col">Note</th>
+  </tr>
+  <tr>
+    <th scope="row" align="left">
+      <a href="https://www.w3.org/2001/tag/doc/polyfills/#encapsulate-as-a-module-or-umd">
+      3.1 Encapsulate as a module or UMD
+      </a>
+    </th>
+    <td align="center">
+      <ul><li>- [x] </li></ul>
+    </td>
+    <td>
+      ES Module
+    </td>
+  </tr>
+  <tr>
+    <th scope="row" align="left">
+      <a href="https://www.w3.org/2001/tag/doc/polyfills/#use-smart-distribution">
+      3.2 Use smart distribution
+      </a>
+    </th>
+    <td align="center">
+      <ul><li>- [x] </li></ul>
```

Comment:
Just noting that - because this looks weird, and is semantically incorrect - for some reason the `- [ ]` markdown checkboxes must seemingly be wrapped in a list, or else they wont render (tried using `<p>` for example, didn't work). And I wanted to use HTML tables to 1) control presentation through HTML attributes and 2) improves readability vs markdown tables.